osmnx/pois.py

```diff
@@ -0,0 +1,414 @@
+################################################################################
+# Module: pois.py
+# Description: Download and plot Points of Interests (POIs) from OpenStreetMap
+# License: MIT, see full license in LICENSE.txt
+# Web: https://github.com/gboeing/osmnx
+################################################################################
+
+from shapely.geometry import Point, LineString, MultiPolygon, Polygon, box
+import geopandas as gpd
+from .core import overpass_request, bbox_from_point, gdf_from_place
+from .utils import log, geocode, bbox_to_poly
+
+def parse_poi_query(west, south, east, north, amenities=None, timeout=180, maxsize=''):
+    """
+    Parse the Overpass QL query based on the list of amenities.
+
+    Parameters
+    ----------
+    west : float
+        Westernmost coordinate of the bounding box of the search area.
+    south : float
+        Southernmost coordinate from bounding box of the search area.
+    east : float
+        Easternmost coordinate from bounding box of the search area.
+    north : float
+        Northernmost coordinate from bounding box of the search area.
+    amenities : list
+        List of amenities that will be used for finding the POIs from the selected area.
+    timeout : int
+        Timeout for the API request. 
+    """
+    if amenities:
+        # Overpass QL template
+        query_template = ('[out:json][timeout:{timeout}]{maxsize};((node["amenity"~"{amenities}"]({south:.8f},'
+                          '{west:.8f},{north:.8f},{east:.8f});(._;>;););(way["amenity"~"{amenities}"]({south:.8f},'
```

Comment:
For consistency, could you round all these coordinates (and below) to 6 decimal places?

Reply:
Done.